### PR TITLE
Video camera tweaks

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Video camera must be in hands to be turned on. Item size is now NORMAL and no longer fits in belt slots.
Advanced video camera added to librarian, can fit and be used in a belt slot.
Video camera redundant description has been removed.
Message about video camera being turned on/off is now visible to make others aware of its presence.
Added video camera as protolathe design (1000 Metal, 500 Glass, programming" = 3, "materials" = 2)
Added in hand sprites for the video camera (sprite isnt good but its better than before (nothing), if a spriter wishes to make a good version for it you are welcome :) ).

## Why It's Good For The Game

I believe that, by forcing it to be in hand, will prevent the old "security with cameras" meta we used to have, wich led to the removal of the video camera from the autolathe.
It's odd to see a shoulder camera being able to fit into belt/pocket slots.
## Images of changes

viideocamera (2)

## Testing

- Did activate the camera while verifying the monitor multiple times.
- Printed it from the protolathe.
- I dropped and stored it multiple times, it always did turn off.
## Changelog
:cl:
tweak: Video camera size has been increased and must in hands to be turned on
imageadd: Added in hand sprites for video camera
spellcheck: Video camera no longer has redundant description
/:cl:

### CODE REVIEW

**Comment by Xsingular** [comment]
That kinda removes one of the few cool things that a Librarian has access to.

Wouldn't it be better to keep the Librarian video camera as it currently exists as a separate item that cannot be made in RnD (nor deconstructed, I guess)? 
For all of those Journalists out there that want to show their viewers something while interacting with the station and/or space, especially when reporting from the "frontlines", while also opening the door for other crew to broadcast anything else with a less specialized set up.

**Comment by Henri215** [comment]
> That kinda removes one of the few cool things that a Librarian has access to.
> 
> Wouldn't it be better to keep the Librarian video camera as it currently exists as a separate item that cannot be made in RnD (nor deconstructed, I guess)? For all of those Journalists out there that want to show their viewers something while interacting with the station and/or space, especially when reporting from the "frontlines", while also opening the door for other crew to broadcast anything else with a less specialized set up.

A unique version for roundstart librarians that isnt two handed and can be used while in hand or inside your belt slot, what do you think? (if possible i wouldnt want a video camera that films stuff from inside a backpack).

In regards to the current state of the PR, if its decided that forcing it to be used only with one hand, instead of two, is enough to prevent abuse then i would happily change it since it was the initial idea. I personally think it would be better to have a librarian with a camera in his hands, to signal to others that he is filming, than some sort of hidden camera.

**Comment by Xsingular** [comment]
> A unique version for roundstart librarians that isnt two handed and can be used while in hand or inside your belt slot, what do you think? (if possible i wouldnt want a video camera that films stuff from inside a backpack).
That sounds amazing, good stuff!
 
At the moment there's very little incentive for most people to play the roles like Librarian or Barber, so giving them this extra cool factor you described would go a long towards giving them a reason to play a proper Journalist!

File: code/modules/paperwork/photography.dm [hunk 1]
```diff
@@ -544,47 +544,44 @@ GLOBAL_LIST_INIT(SpookyGhosts, list("ghost","shade","shade2","ghost-narsie","hor
...
-		else
-			src.icon_state = icon_on
-			camera.network = list("news")
-			camera.c_tag = user.name
-	else
+/obj/item/twohanded/videocam/wield(mob/living/carbon/user)
+	. = ..()
+	on = TRUE
+	camera = new /obj/machinery/camera(src)
+	src.icon_state = icon_on
```
Comment (Edan52) [inline code/modules/paperwork/photography.dm:565]: src should be implied here

File: code/modules/paperwork/photography.dm [hunk 2]
```diff
@@ -544,47 +544,44 @@ GLOBAL_LIST_INIT(SpookyGhosts, list("ghost","shade","shade2","ghost-narsie","hor
...
-		camera = new /obj/machinery/camera(src)
-		camera.network = list("news")
-		GLOB.cameranet.removeCamera(camera)
-		camera.c_tag = user.name
-	to_chat(user, "You switch the camera [on ? "on" : "off"].")
+/obj/item/twohanded/videocam/unwield(mob/living/carbon/user)
+	. = ..()
+	if(on)
+		on = FALSE
+		src.icon_state = icon_off
```
Comment (Edan52) [inline code/modules/paperwork/photography.dm:574]: here too, might also want to make this early return as well

**Comment by Henri215** [comment]
Added librarian version of the video camera, can fit into belts and used in it (no different sprite for it, maybe a brown strip? idk).
Message about the camera being toggled on/off is now visible  to make others aware of its presence.

File: code/game/jobs/job/support.dm [hunk 3]
```diff
@@ -433,7 +433,7 @@
 	l_hand = /obj/item/storage/bag/books
 	pda = /obj/item/pda/librarian
 	backpack_contents = list(
-		/obj/item/videocam = 1)
+		/obj/item/videocam/videocam_adv = 1)
```
Comment (SteelSlayer) [inline code/game/jobs/job/support.dm:436]:
```suggestion
		/obj/item/videocam/advanced = 1)
```

**Review by SteelSlayer — COMMENTED** [review]
Some nitpicks, otherwise GTG

File: code/game/jobs/job/support.dm (hunk 3 above)
Comment (Henri215) [reply code/game/jobs/job/support.dm:436]: added in next commit

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**